plugins/modules/cloudwatchevent_rule.py

```diff
@@ -204,6 +204,19 @@ def _format_json(json_string):
         return str(json.dumps(json_string))
 
 
+def _validate_json(s):
+    try:
+        json.loads(s)
+        return True
+    except json.JSONDecodeError:
+        return False
+
+
+def _snakify(dict):
```

Comment:
Instead of moving this here, could we just call `camel_dict_to_snake_dict` directly wherever this is called?

Reply:
Agree!

Reply:
removed _snakify()